src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaVersionUpload/tests/MediaVersionUpload.test.js

```diff
@@ -36,40 +36,32 @@ jest.mock('../../../stores/MediaFormatStore', () => jest.fn(function() {
     this.loading = false;
 }));
 
-test('Render a loading MediaVersionUpload field', () => {
-    const resourceStore = new ResourceStore('media', 4, {locale: observable.box('de')});
-    resourceStore.loading = true;
-
-    expect(render(
-        <MediaVersionUpload
-            resourceStore={resourceStore}
-        />
-    )).toMatchSnapshot();
-});
-
-test('Render a non loading MediaVersionUpload field', () => {
+test('Render a MediaVersionUpload field', () => {
     const resourceStore = new ResourceStore('media', 4, {locale: observable.box('de')});
     resourceStore.loading = false;
 
     expect(render(
         <MediaVersionUpload
+            onSuccess={jest.fn()}
             resourceStore={resourceStore}
         />
     )).toMatchSnapshot();
 });
 
-test('Should update resourceStore after SingleMediaUpload has completed upload', () => {
+test('Should update resourceStore and call onSuccess after SingleMediaUpload has completed upload', () => {
     const testFile = {name: 'test.jpg'};
     const resourceStore = new ResourceStore('media', 4, {locale: observable.box('de')});
     resourceStore.loading = false;
 
     const mediaVersionUpload = mount(<MediaVersionUpload
+        onSuccess={jest.fn()}
```

Comment:
Move this to a variable called `successSpy`, and use that one to assert if it has been called at the end of the test. This way it does not matter who get the `onSuccess` prop, and will work regardless of any implementation details.